TurkishFilter/sections/specific.txt

```diff
@@ -68,6 +68,7 @@ realfilmizle3.com,fullhdfilmizlett.com,fullhdfilmcibaba1.com,dizimax1.com,dizifi
 /service/advertchannels?$xmlhttprequest,redirect=nooptext,domain=mardinhaber.com.tr|gidahatti.com|bolugundem.com
 dirilispostasi.com,turkuazgazetesi.net,yenialanya.com,bugunkocaeli.com.tr,cagdaskocaeli.com.tr,mansetaydin.com,seskocaeli.com,kocaelibarisgazetesi.com,kocaelikoz.com,enkocaeli.com##.block-pano > div.us
 !
+dizikorea.com###router-view > .player-top-ad > div[style="width: 728px; height: 90px; display: inline-block;position: relative;"]
```

Comment:
Why not `dizikorea.com##.player-top-ad`?

Reply:
I would suggest reducing this rule to:
`dizikorea.com##.player-top-ad > div[style*="height: 90px;"]`

Reply:
I would suggest reducing this rule to:
`dizikorea.com##.player-top-ad > div[style*="height: 90px;"]`